### ISSUE TITLE
core: we no longer anticipate the speed limits

### ISSUE THREAD

**Issue by Castavo** [OpenRailAssociation/osrd#10136]
### What happened?

Trains respect speed limit at the closest.

### What did you expect to happen?

Trains should respect lowering of speed limits 100m before their actual start.

This was present in the version of simulations for train schedule V1, and was dropped.

As seen with @Arthur-Lefebvre, @axrolld and @Tguisnet, it is expected that simulations apply this 100m margin.

### How can we reproduce it (as minimally and precisely as possible)?

1. Make a simulation and witness that the breaking curves end right to the speed limit decreases

### On which environments the bug occurs?

Recette (SNCF)

### On which browser the bug occurs?

Firefox

### OSRD version (top right corner `Account` button > `Informations`)

6580cfe

**Comment by eckter** [comment]
So according to @loic-hamelin, we don't actually want this in the current running time computations. It's not relevant now and we'll only think about this with the v3. Apparently it's wrong to apply it in TVM or ERTMS contexts.

@Castavo  @Arthur-Lefebvre, @axrolld and @Tguisnet: does it sound good to you? would you be OK with me reverting https://github.com/OpenRailAssociation/osrd/pull/10143?

**Comment by Arthur-Lefebvre** [comment]
I agree with the fact that it doesn't apply in TVM and ERTMS contexts, but it applies in all others.

When I have to modelize it, I treat this as a driver behaviour. It's usual that the driver behaviour depends of the signalling system.

If it's not possible to make th difference now and I had to choose, I'd rather keep this PR but both are possible.

**Comment by Castavo** [comment]
I don't care whether we keep it or not. 
Maybe we should ask some of our clients first ? We'll discuss about this and come back to you, thanks for letting us know
